assets/airports/kstl.json

```diff
@@ -1246,7 +1246,7 @@
             "origin": "kstl",
             "destination": "",
             "category": "departure",
-            "route": "KSTL.LINDY6.ARG",
+            "route": "KSTL29.LINDY6.ARG",
```

Comment:
29 or 30L? If you have to turn left to STL to continue en-route, this would make sense on 30L unless you can fly direct to join the transition